nautobot/core/templatetags/helpers.py

```diff
@@ -616,6 +616,48 @@ def render_uptime(seconds):
     )
 
 
+@library.filter()
+@register.filter()
+def hyperlinked_field(value, hyperlink):
+    """Render a value as a hyperlink."""
+    if not value:
+        return placeholder(value)
+    return format_html('<a href="{}">{}</a>', value, hyperlink)
+
+
+@library.filter()
+@register.filter()
+def render_m2m(value):
```

Comment:
Is this specific to content-types or is it actually applicable to M2M in general?

Reply:
ContentTypes